tutorials/quick_start/common/results.py

```diff
@@ -109,7 +112,9 @@ def parse_results(params: dict, float_acc: float, quant_acc: float, quant_info:
     """
     a_bits = quant_info.tpc_info['Target Platform Model']['Default quantization config']['activation_n_bits']
     w_bits = quant_info.tpc_info['Target Platform Model']['Default quantization config']['weights_n_bits']
-    bit_config = 'W' + str(w_bits) + 'A' + str(a_bits)
+    bit_config = f'W{str(w_bits)}A{str(a_bits)}'
```

Comment:
I don't think that you need the `str()` here if you used `f' '` form, it does the casting automatically